src/core/i18n/en/translation.en.json

```diff
@@ -867,14 +867,14 @@
     "draftNotice": "DRAFT - Only visible for $t(common.admins)",
     "closed": "This discussion is closed so you can no longer comment",
     "notMember": "You can't reply to this discussion since you're not a $t(common.member) of this $t(common.space)",
-    "delete-confirm-title": "Delete Callout?",
-    "delete-confirm-text": "All $t(common.posts), $t(common.whiteboards) and messages under the Callout will also be deleted.",
+    "delete-confirm-title": "Delete $t(common.post)?",
```

Comment:
hmmm careful with this, I know we are tending to call "Callouts" => "Posts" or "Collaboration Tools", but maybe in the future we rename "Posts" to "Cards" or something like that... whatever is internally called callout should remain as callout I think. I would change the content of `common.callout` to "Post", that I'm seeing it's not used a lot and see the areas affected and use here `$(common.callout)`